src/main/java/com/alkemy/ong/controller/UserController.java

```diff
@@ -0,0 +1,24 @@
+package com.alkemy.ong.controller;
+
+import com.alkemy.ong.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping(value = "users")
+public class UserController {
+
+    private final UserService userService;
```

Comment:
agregar autowired al service

Reply:
Hola, se utiliza lo que se conoce [constructor injection](https://reflectoring.io/constructor-injection/), no es necesario autowired. Porque se utiliza `@AllArgsConstructor` ya que segun el [IDE es no recomendable utilizar Autowired ](https://stackoverflow.com/questions/62845494/autowired-says-field-injection-not-recommended)en un Field Injection con Autowired

Reply:
Perfecto!